docs/manual-CN/5.appendix/gremlin-ngql.md

```diff
@@ -92,125 +92,341 @@ gremlin> g.V(jesus).property('age', 6000);
 
 - 查看数据
   
-  ```bash
-  nebula> FETCH PROP ON character hash("saturn");
-  ===================================================
-  | character.name | character.age | character.type |
-  ===================================================
-  | saturn         | 10000         | titan          |
-  ---------------------------------------------------
-
-  gremlin> g.V(saturn).valueMap();
-  ==>[name:[saturn],type:[titan],age:[10000]]
-  ```
+```bash
+nebula> FETCH PROP ON character hash("saturn");
+===================================================
+| character.name | character.age | character.type |
+===================================================
+| saturn         | 10000         | titan          |
+---------------------------------------------------
+
+gremlin> g.V(saturn).valueMap();
+==>[name:[saturn],type:[titan],age:[10000]]
+```
 
 - 查询 hercules 的祖父
 
-    ```bash
-    nebula> GO 2 STEPS FROM hash("hercules") OVER father YIELD  $$.character.name;
-    =====================
-    | $$.character.name |
-    =====================
-    | saturn            |
-    ---------------------
-
-    gremlin> g.V().hasLabel('character').has('name','hercules').out('father').out('father').values('name');
-    ==>saturn
-    ```
+```bash
+nebula> GO 2 STEPS FROM hash("hercules") OVER father YIELD $$.character.name;
+=====================
+| $$.character.name |
+=====================
+| saturn            |
+---------------------
+
+gremlin> g.V().hasLabel('character').has('name','hercules').out('father').out('father').values('name');
+==>saturn
+```
 
 - 查询 hercules 的父亲
 
-    ```bash
-    nebula> GO FROM hash("hercules") OVER father YIELD $$.character.name;
-    =====================
-    | $$.character.name |
-    =====================
-    | jupiter           |
-    ---------------------
-
-    gremlin> g.V().hasLabel('character').has('name','hercules').out('father').values('name');
-    ==>jupiter
-    ```
+```bash
+nebula> GO FROM hash("hercules") OVER father YIELD $$.character.name;
+=====================
+| $$.character.name |
+=====================
+| jupiter           |
+---------------------
+
+gremlin> g.V().hasLabel('character').has('name','hercules').out('father').values('name');
+==>jupiter
+```
 
 - 查询年龄大于 100 的人物
 
-    ```bash
-    nebula> XXX # coming soon
-    gremlin> g.V().hasLabel('character').has('age',gt(100)).values('name');
-    ==>saturn
-    ==>jupiter
-    ==>neptune
-    ==>pluto
-    ```
+```bash
+nebula> LOOKUP ON character WHERE character.age > 100 YIELD character.name AS name;
```

Comment:
can you print out the result?

Reply:
Of course, I didn't because this will make a really long doc. I'll give the results if you think they are necessary.